### ISSUE TITLE
Function createPasswordReset in activator.js throw TypeError: Object #<ServerResponse> has no method 'sendStatus'

### ISSUE THREAD

**Issue by flexpert** [deitch/activator#22]
Hi, 

I'm using ```javascript"activator": "^1.0.1"``` on package.json file for activating users.

On the reset password flow I'm getting this error:
```javascriptTypeError: Object #<ServerResponse> has no method 'sendStatus'```

It comes from this function:
```javascript
createPasswordReset: function (req,res,next) {
	rparam(req);
	createPasswordReset(req,function (code,message) {
		if (message === null || message === undefined || (typeof(message) === "number" && message === code)) {
			res.sendStatus(code);
		} else {
			res.status(code).send(message);
		}
	});
}`
```

The problem is on ```res.sendStatus(code);```, am I missing something here? 
As far I know that should be ```res.status(code);```, like in all other places.

Is it yours or mine bug :smirk:?

Thanks

**Comment by deitch** [comment]
Which version of expressjs are you using? See https://github.com/deitch/activator/blob/master/README.md#versions

* activator >=1.0.0 only supports express >=4.0 which has `res.sendStatus()` see http://expressjs.com/4x/api.html#res.sendStatus
* activator <1.0.0 only support express <4.0 which does not

Is that it?

**Comment by flexpert** [comment]
Yeah, That was it...
I used ```express: ~4.0.0``` in ```packages.json```, which resolved to ```3.9.x```, 
And that's what caused the problem

Thanks
